qcodes/tests/dataset/test_descriptions.py

```diff
@@ -131,14 +134,15 @@ def test_default_dictization_for_storage(some_interdeps):
     idps_old = new_to_old(idps_new)
 
     new_desc = RunDescriber(idps_new)
-    old_desc = {'version': 2,
+    old_desc = {'version': 3,
                 'interdependencies': idps_old._to_dict(),
-                'interdependencies_': idps_new._to_dict()}
+                'interdependencies_': idps_new._to_dict(),
+                'shapes': None}
 
     assert serial.to_dict_for_storage(new_desc) == old_desc
 
 
-def test_dictization_of_version_2(some_interdeps):
+def test_dictization_of_version_3(some_interdeps):
```

Comment:
no more testing the test_dictization_of_version_2 ? :)

Reply:
We could add an explicit test for this but the change of the name for this one indicates that the name was wrong to begin with.

Reply:
do you mean that the name should be "dictization of current version"?

Reply:
yes, there is not really any such thing as a older version rundescriber any more just the dict and disc representation